pkg/config/config.go

```diff
@@ -586,6 +586,18 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+func (c *EngineConfig) findRuntime() string {
+	// Search for crun first followed by runc and kata
+	for _, name := range []string{"crun", "runc", "kata"} {
+		for _, v := range c.OCIRuntimes[name] {
```

Comment:
Suggestion: if all of this fails we should search $PATH

Reply:
Do you mean search crun path before or after it searches for runc and kata paths?

Reply:
Searching `crun` before moving on seems better, but either works, as long as we do check $PATH eventually.

Reply:
+1 for checking `crun` on `PATH` beforehand.
re https://github.com/containers/podman/blob/v2.2.1/libpod/runtime.go#L157 :
With the current logic, if `runc` is at one of the predefined paths and `crun` is not (but can be found at another location on `PATH`), `runc` would be picked even if cgroups v2 are used. If "`crun` on `PATH`" is not checked beforehand, additional cgroups v2-logic might be needed, AFAIUI.